release-1.8/release_notes_draft.md

```diff
@@ -203,6 +204,18 @@ kind.
   metrics) are graduating to v1beta1.  See [Action Required Before
   Upgrading](#action-required-before-upgrading).
 
+* metrics-server is now the reccomended way to provide the resource
+  metrics API. It is deployable as an addon, similarly to how Heapster is
```

Comment:
similar to

Reply:
We're not describing the metrics server itself.  Rather, we're describing the act of deploying it, so I believe we should use the adverbial form (the actions used to deploy heapster are similar to those used to deploy metrics server, so we say that you deploy the two similarly).  For example, you'd say "it's deployable easily", not "it's deployable easy".  Same here, except "similarly" instead of "similar".